internal/validation/validation_utils.go

```diff
@@ -120,6 +141,20 @@ func ValidateIPv4AddressSchema(i interface{}, k string) (_ []string, errors []er
 	}
 }
 
+func ValidateCidrIPv4AddressSchema(i interface{}, k string) (_ []string, errors []error) {
```

Comment:
Can you, please, add unit test for this method?

Reply:
Done